### PR TITLE
Feature/gemmolo 74

### PR DESCRIPTION
**Pull request by CSergienko**
Form styling has been implemented, please have a look.

### CODE REVIEW

File: gem/static/css/_style.surveys.scss [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    width: 100%;
+}
+
+%surveys__checkbox {
+    // display: inline-block;
+    // width: auto;
+    // height: auto;
+    // margin: 0 16px 0 0;
+    // margin: 0 1rem 0 0;
+    // vertical-align: middle;
```
Comment (Pringels) [inline gem/static/css/_style.surveys.scss]: Do you still need these?

File: gem/static/css/_style.surveys.scss [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            width: 16px;
+            width: 1rem;
+            height: 16px;
+            height: 1rem;
+            margin-right: 16px;
+            margin-right: 1rem;
+            padding: 4px;
+            padding: 0.25rem;
+            background-color: #ffffff;
+            border: 1px solid #666666;
```
Comment (Pringels) [inline gem/static/css/_style.surveys.scss]: Maybe make a colour variable?

**Comment by Pringels** [comment]
+1

File: gem/static/css/_style.surveys.scss [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+    &__helptext {
+        display: block;
+        font-size: 16px;
+        font-size: 1rem;
+    }
+
+    &__form {
+        padding: 24px 16px;
+        padding: 1.5rem 1rem;
```
Comment (markupguy) [inline gem/static/css/_style.surveys.scss]: our of interest, why not use an automated fallback?

**Comment by markupguy** [comment]
+1

File: gem/static/css/_style.surveys.scss (hunk 1 above)
Comment (CSergienko) [inline gem/static/css/_style.surveys.scss]: No, will remove.

File: gem/static/css/_style.surveys.scss (hunk 3 above)
Comment (CSergienko) [inline gem/static/css/_style.surveys.scss]: The SCSS gets built on the server side, so I can't do any pre or post processing on the files without digging into the guts of it. I'm considering killing the fallback entirely, as it doesn't exist anywhere else I can see in the project.

**Pull request merged**